docs/source/en/api/pipelines/aura_flow.md

```diff
@@ -62,6 +62,34 @@ image = pipeline(prompt).images[0]
 image.save("auraflow.png")
 ```
 
+Loading [GGUF checkpoints](https://huggingface.co/city96/AuraFlow-v0.3-gguf) are also supported:
+
+```py
+import torch
+from diffusers import (
+    AuraFlowPipeline,
+    GGUFQuantizationConfig,
+    AuraFlowTransformer2DModel,
+)
+
+transformer = AuraFlowTransformer2DModel.from_single_file(
+    "https://huggingface.co/city96/AuraFlow-v0.3-gguf/blob/main/aura_flow_0.3-Q2_K.gguf",
+    quantization_config=GGUFQuantizationConfig(compute_dtype=torch.bfloat16),
+    torch_dtype=torch.bfloat16,
+    config="fal/AuraFlow-v0.3",
```

Comment:
```suggestion
```

`default_subfolder` is only used in the config detection path so this will fail, if we're providing `config` we should also pass `subfolder` or just remove `config` as detection is working.